proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -13427,14 +13427,80 @@ function longFromRange( test )
   });
 
 };
+
 //
 
-function longFromRange1( test )
+function longFromRangeWithStep( test )
 {
+  test.case = 'equal numders of range';
```

Comment:
Descriptions of test cases should have same structure and it should describe difference between cases. 

For example, this test case shoul has minimum 3 variants. It is little group of test cases:

test.case = 'range - equal numbers, step - positive number';
// code
test.case = 'range - equal numbers, step - negative number';
// code
test.case = 'range - equal numbers, step - undefined';
// code

Also, equal numbers in range might be described like 
'range[ 0 ] === range[ 1 ]' 

// 

So, next group of test has description like
test.case = 'range[ 0 ] < range[ 1 ], .........`